x/tx/signing/direct_aux/direct_aux.go

```diff
@@ -50,7 +50,11 @@ func NewSignModeHandler(options SignModeHandlerOptions) SignModeHandler {
 	}
 
 	if options.SignersContext == nil {
-		h.signersContext = signing.NewGetSignersContext(signing.GetSignersOptions{ProtoFiles: h.fileResolver})
+		var err error
+		h.signersContext, err = signing.NewGetSignersContext(signing.GetSignersOptions{ProtoFiles: h.fileResolver})
+		if err != nil {
+			panic(err)
```

Comment:
I think the signature of `NewSignModeHandler` should be changed to return err, it probably should have been from the start.  I can fix this in a follow up PR if you don't get to it.

Reply:
Fixed in this PR.